bangc-ops/mlu_op.h

```diff
@@ -13711,6 +13727,158 @@ mluOpDiffIouRotatedSortVerticesForward(mluOpHandle_t handle,
                                        const mluOpTensorDescriptor_t idx_desc,
                                        void *idx);
 
+// Group:RoiPoolingForward
+/*!
+ * @brief Generates fixed size feature map and input feature index
+ * of argmax for each Roi(Regions of Interest) to perform ::mluOpRoiPoolingForward operation.
+ *
+ * @param[in] handle
+ * Handle to an MLUOP context that is used to manage MLU devices and queues in
+ * ::mluOpRoiPoolingForward operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] pooling_mode
+ * The pooling mode of roipoolingforward defined in ::mluOpPoolingMode_t.
+ * @param[in] input_desc
+ * The descriptor of the input tensor in the roipoolingforward process. For detailed
+ * information, see ::mluOpTensorDescriptor_t.
+ * @param[in] input
+ * Pointer to the MLU memory that stores the input tensor.
+ * @param[in] rois_desc
+ * The descriptor of rois tensor. For detailed information, see ::mluOpTensorDescriptor_t.
+ * @param[in] rois
+ * Pointer to the MLU memory that stores the rois tensor. Rois means regions of interest.
+ * @param[in] spatial_scale
+ * The spatial scale of each regions of interest in the input feature map.
+ * @param[in] output_desc
+ * The descriptor of output tensor. For detailed information, see ::mluOpTensorDescriptor_t.
+ * @param[out] output
+ * Pointer to the MLU memory that stores the output tensor.
+ * @param[out] argmax
+ * Pointer to the MLU memory that stores the argmax tensor. Pointer may be NULL. Argmax tensor
+ * means input feature index of maximum for each Roi(Regions of Interest).
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM.
+ *
+ * @par Data Type
+ * - This function supports the following data types for input tensor \b input, rois tensor \b rois,
+ *   output tensor \b output, and argmax tensor \b argmax. The data type of the \b input \b rois \b output
+ *   should be the same. The data type of the argmax is different from others.
```

Comment:
The data type of the argmax is different from others. 感觉这句可以删掉呀
```suggestion
 *   must be the same. The data type of the argmax is different from others.
```